Documentation/minio-object-store.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Minio Object Store
+weight: 38
```

Comment:
this value needs to be in the 20's to fit with the other user guides in the navigation. 28 should work I think so it will go after these ones:
https://github.com/rook/rook/blob/master/Documentation/block.md
https://github.com/rook/rook/blob/master/Documentation/object.md
https://github.com/rook/rook/blob/master/Documentation/filesystem.md